core/vm/runtime/runtime.hpp

```diff
@@ -253,43 +253,43 @@ namespace fc::vm::runtime {
     template <typename M>
     outcome::result<typename M::Result> sendM(const Address &address,
                                               const typename M::Params &params,
-                                              TokenAmount value) {
+                                              const TokenAmount &value) {
       OUTCOME_TRY(params2, actor::encodeActorParams(params));
       OUTCOME_TRY(result, send(address, M::Number, params2, value));
       return actor::decodeActorReturn<typename M::Result>(result);
     }
 
     /// Send funds
-    inline auto sendFunds(const Address &to, BigInt value) {
+    inline auto sendFunds(const Address &to, const BigInt &value) {
       return send(to, kSendMethodNumber, {}, value);
     }
 
     /// Send with typed result R
     template <typename R>
-    outcome::result<R> sendR(Address to_address,
+    outcome::result<R> sendR(const Address &to_address,
                              MethodNumber method_number,
                              const MethodParams &params,
-                             BigInt value) {
+                             const BigInt &value) {
```

Comment:
The same changes for send() method on line 126.
And maybe we want to use TokenAmount instead of BigInt here